Sources/ApodiniDatabase/PropertyVisitors/UpdatableFieldProperty.swift

```diff
@@ -0,0 +1,37 @@
+import Fluent
+
+/// A protocol all field property visitors that want to update values of a `FieldProperty` have to conform to.
+/// It returns whatever has been specified as `Value`.
+protocol UpdatableFieldPropertyVisitor where Value: Codable {
+    associatedtype Value
+
+    func visit<Model, V>(_ property: inout FieldProperty<Model, V>) -> Value
+}
+
+/// A protocol to make the `FieldProperty` of a database model updatable at run time.
+/// It returns `true` if the update of the given `FieldProperty` was successful, else `false`
+protocol UpdatableFieldProperty {
+    func accept<Visitor: UpdatableFieldPropertyVisitor>(_ visitor: Visitor) -> Visitor.Value where Visitor.Value == Bool
+}
```

Comment:
As far as I can see there is no other implementation then the one defined in this scope and all use `Bool` as the concrete type for `Value`. Therefore I would suggest to remove the `associatedtype` completely and only use concrete types.